server/webpa.go

```diff
@@ -155,6 +163,11 @@ func (b *Basic) New(logger log.Logger, handler http.Handler) *http.Server {
 				ClientAuth: tls.RequireAndVerifyClientCert,
 			}
 			tlsConfig.BuildNameToCertificate()
+
+			// Add verify peer ceritifcate callback for additional validation
+			if b.PeerVerify && b.PeerVerifyFunc != nil {
+				tlsConfig.VerifyPeerCertificate = *b.PeerVerifyFunc
```

Comment:
There's no need to dereference `PeerVerifyFunc`.  Functions are reference types in `golang`